apps/sms/views/conversation/style/composer.css

```diff
@@ -4,12 +4,6 @@
   background: #e1f0ec;
```

Comment:
Yes, that's my original idea and it looks reasonable to keep these in view related css file, but I'm not sure which file is more appropriate so I didn't move first. But maybe even sms.css is better than composer.css here.